plugins/sniff.yaml

```diff
@@ -0,0 +1,53 @@
+apiVersion: krew.googlecontainertools.github.com/v1alpha2
+kind: Plugin
+metadata:
+  name: sniff
+spec:
+  version: "v1.0.0"
```

Comment:
This should be 1.1.0 but it’s optional. Feel free to drop it.

Reply:
changed to 1.1.0